### ISSUE TITLE
ThermalExpansion has no miniumum required version of codechicken lib

### ISSUE THREAD

**Issue by mezz** [CoFH/Feedback#9]
Without the dependency version info the game crashes when you have the wrong version, instead of FML showing a screen saying "hey you need CCL version ____".
https://gist.github.com/mezz/aa689453ed078774ef8512cb9f5d4496

Something like `"required-after:CodeChickenLib@[2.5.6.233,)"`

**Comment by KingLemming** [comment]
Yeah, that's known but couldn't actually be handled in dev due to some dumb reasons. Will be fixed. :)
